src/classes/ERR_ExceptionHandler_TEST.cls

```diff
@@ -83,5 +83,15 @@ private with sharing class ERR_ExceptionHandler_TEST {
 		ERR_ExceptionHandler.beautifyExceptionMessage(ex);
 
 		system.assertEquals(oldMessage, ex.getMessage(), 'Unhandled exception type should be unmodified.');
-	}
+    }
+    
+    @isTest
+    private static void testAffilMapper() {
+        AFFL_MultiRecordTypeMapper afflMapper1 = ERR_ExceptionHandler.affilMapper;
```

Comment:
@kevinswiggum This unit test validates that the caching is working appropriately. Our assumption for this fix is that the first call will create an object and return it and that subsequent calls in the same transaction will return that exact object in order to avoid the constructor running another SOQL query. It also performs an additional validation that the caching is avoiding any additional SOQL queries. If this unit test breaks, then it would serve as an appropriate warning that the caching is no longer serving its intended purpose and that the changes that caused it to break could constitute a breaking change that would reintroduce the bug.

We did miss the mark on this one slightly by not making the property final to further enforce that.